src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
```

Comment:
The 'margin-top: 0;' style for the body seems unnecessary as the default browser style for the body element doesn't usually have any margin-top. You can remove this line to keep the CSS clean. 🧹

Reply:
Hi!
thanks for the tip, i'll edit it now
but my chrome default top margin is 8px
I decided to reset it just in case

![image](https://user-images.githubusercontent.com/127268424/231426292-811b008e-c640-4849-a49c-959ada89ea04.png)

Reply:
Can you give me a hint, please.
I made changes and deployed.
Do I need to do something else with it?

Reply:
nope, if tests and code are ok, you're done

Reply:
let's wait for mentors to review the task more precisely, I'm here to test things 🍃

Reply:
thanks )) have a nice day